cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -908,8 +908,9 @@ function(install_celix_bundle_targets)
 
     get_target_property(EXPORT_BUNDLES celix-bundles EXPORT_${EXPORT_NAME}_BUNDLES)
 
-    if (NOT DEFINED EXPORT_BUNDLES)
-        message(FATAL_ERROR "Export ${EXPORT_NAME} not defined. Did you forgot to use a install_celix_bundle with the 'EXPORT ${EXPORT_NAME}' option?")
+    if (NOT DEFINED EXPORT_BUNDLES OR NOT EXPORT_BUNDLES)
```

Comment:
Now that it is possible to turn off all bundles, this is not FATAL error any more.